i18n/en.json

```diff
@@ -1464,6 +1464,18 @@
     "id": "api.outgoing_webhook.disabled.app_error",
     "translation": "Outgoing webhooks have been disabled by the system admin."
   },
+  {
+    "id": "api.plugin.install.download_failed.app_error",
+    "translation": "An error occurred while downloading the plugin."
+  },
+  {
+    "id": "api.plugin.install.insecure_url.app_error",
+    "translation": "An insecure url was given to download the plugin. Please provide a secure url or enable PluginSettings.AllowInsecureDownloadUrl in your configuration."
```

Comment:
For purposes of error, is this checking for HTTPS in the URL or is it actually checking if the site is secure, etc.  Could this error also be thrown if the certificsate is bad for example?

Reply:
At the moment, it is only checking for HTTPS in the URL.

The eventual goal is to verify that a certified plugin is correctly signed, once support of signing plugins is implemented. So it would check for HTTPS, along with checking that the signature is correct. In order to get around the signature checks, the config value would need to be set to false.

Checking if the SSL cert is valid is an interesting idea. I'm not sure if there are spots throughout the application that check for valid certs. @crspeller do you have any thoughts on this?

Reply:
Thanks!  Was just asking, we don't need to check for SSL ceertsI don't think.  Was just wondering what level of check was being performed so the error is 'correct' to the user.

Reply:
The checking of TLS certs should be automatic and controlled by the EnableInsecureOutgoingConnections setting once you move to using the httpservice client.